.github/google_java_format.yml

```diff
@@ -3,8 +3,8 @@ name: Format
 
```

Comment:
Finally, found out why the actions are not getting triggered. The yml file should be in `.github/workflows/`